templates/common-workspaces.tpl

```diff
@@ -0,0 +1,33 @@
+{{- /*
+
+  Contains the results produced by buildah, should be part of a Tasks ".spec.results[]".
+
+*/ -}}
+{{- define "common_workspaces" -}}
```

Comment:
For the `git-clone` `Task`, the "main" workspace needs to be called `output` in order to be backward compatible.

Reply:
cool, will do the changes then.